custom/icds_reports/tasks.py

```diff
@@ -51,6 +51,31 @@
 _dashboard_team_soft_assert = soft_assert(to=DASHBOARD_TEAM_EMAILS)
 
 
+UCR_TABLE_NAME_MAPPING = [
+    {'type': "awc_location", 'name': 'static-awc_location'},
+    {'type': 'awc_mgmt', 'name': 'static-awc_mgt_forms'},
+    {'type': 'ccs_record_monthly', 'name': 'extended_ccs_record_monthly_tableau'},
+    {'type': 'child_health_monthly', 'name': 'extended_child_health_monthly_tableau'},
```

Comment:
These cause tests to fail locally. These don't actually exist and are only on the QA domain.